### ISSUE TITLE
setup.py auto completion not working for python3?

### ISSUE THREAD

**Issue by xuhdev** [zsh-users/zsh-completions#330]
If I run `python setup.py <Tab>`, things work normally. However, if I run `python3 setup.py <Tab>`, the autocompletion does not work.

**Comment by hhatto** [comment]
@xuhdev 
sorry for late reply.

my enviroment working for setup.py completion (python and python3).
```
$ head ~/.zsh/completions/_setup.py
#compdef setup.py
# ------------------------------------------------------------------------------
# Description
# -----------
#
#  Completion script for setup.py (http://docs.python.org/distutils/).
#
# ------------------------------------------------------------------------------
# Authors
# -------
```

Could you a more environment information. (OS, zsh version, result for `cat _setup.py`)

**Comment by xuhdev** [comment]
My OS is Debian Jessie, zsh version 5.0.7 x86_64

My _setup.py file is available [here](https://gist.github.com/xuhdev/36153d2ca33abe235b12).

**Comment by xuhdev** [comment]
This issue was resolved after upgrading zsh to 5.2
